project/ws/app/src/lib/routes/network-v2/routes/network-my-connection/network-my-connection.component.html

```diff
@@ -23,16 +23,16 @@
         </div>
     </div>
     <div class="flex flex-1 mb-4">
-        <mat-card class="flex-1 remove-botton-radius min-height">
+        <mat-card class="flex-1 remove-botton-radius">
             <mat-card-header>
                 <mat-card-title class="w-full margin-l-0 f-medium">
-                    <h2 *ngIf="data && data.length && !enableSearchFeature && data.length > 1 ">{{data.length}}
+                    <h2 *ngIf="filterdData && filterdData.length && !enableSearchFeature && filterdData.length > 1 ">{{filterdData.length}}
```

Comment:
same as above , for all instance

Reply:
**filterdData** is alreay defined as array. I think no need to do the optional check again.